firefox-ios/Client/TabManagement/TabManager.swift

```diff
@@ -21,11 +21,12 @@ protocol TabManager: AnyObject {
     var tabs: [Tab] { get }
     var count: Int { get }
     var selectedTab: Tab? { get }
+    var selectedTabUUID: UUID? { get }
     var backupCloseTab: BackupCloseTab? { get set }
     var backupCloseTabs: [Tab] { get set }
-    var normalTabs: [Tab] { get }
+    var normalTabs: [Tab] { get } // Includes active and inactive tabs
     var normalActiveTabs: [Tab] { get }
-    var inactiveTabs: [Tab] { get }
+    var normalInactiveTabs: [Tab] { get }
```

Comment:
Renamed for clarity since private tabs can't be inactive technically, or at least not as we are using "inactive". :)

Reply:
Kind of a nit, but I vote keeping this to `inactiveTabs` because there's a lot of references to "inactiveTabs" without the normal modifier in the codebase that it might actually just be more mental overhead to change the name at this point. Maybe a comment here stating that private tabs can't be inactive would be better?

Reply:
I can revert this pretty easily for now if we want to have a larger discussion on the value of this change!

I agree, other naming should be updated as well in this case. It could be a separate task if we want to go forward with a change like this.

For me, it was the lack of symmetry between `normalActiveTabs` and `inactiveTabs` that was a bit confusing. And that an "active tab" is assumed normal and not private, while an "inactive tab" is assumed also normal and not private. But a "private tab" can't be "inactive tab"—except actually, it can be, we just don't differentiate in the UI or have special private inactive tab logic haha. It starts to get a little confusing to know exactly what each subarray contains as there are now 2 flags at play. 😂 

While I was writing the tests I had to be very explicit with this, which was why I thought the naming could maybe be improved across the three tab types we really care about: normal active tabs, normal inactive tabs, and private tabs (technically either active or inactive).

Reply:
Yeah maybe revert for now and lets have a larger discussion - I agree that something is not quite right with the naming for the tabs. Maybe we can discuss in the iOS devs channel and make a ticket to change the naming from that? You could basically just put what you wrote here.
If we're gonna fix it, we might as well fix it everywhere in the project 😆

Reply:
Sounds good to me! I've started a discussion in slack and made a [proposal task](https://mozilla-hub.atlassian.net/browse/FXIOS-10089) for us to discuss! 👍  Will revert this change in the meantime.